RELEASE-NOTES.txt

```diff
@@ -14,6 +14,7 @@ Unreleased
 * [***] Full-width and wide alignment support for Columns
 * [***] New Block: File [https://github.com/wordpress-mobile/gutenberg-mobile/pull/2835]
 * [**] Image block - Add link picker to the block settings and enhance link settings with auto-hide options [https://github.com/WordPress/gutenberg/pull/27292]
+* [**] Add Support to Insert Existing Reusable Blocks from the Editor.
```

Comment:
I saw in a P2 post that the changes should be added in the [CHANGELOG](https://github.com/WordPress/gutenberg/blob/master/packages/react-native-editor/CHANGELOG.md) file in `Gutenberg` repository instead of here, right?
If that’s the case I’ll add this change in the `Gutenberg` PR so in the new PR for `Gutenberg-mobile` only the submodule reference update will be included.

Reply:
True, I'd overlooked that, thanks for noticing!